x-aeson/src/X/Data/Aeson.hs

```diff
@@ -19,6 +19,7 @@ import           Data.Text
 import           Data.Text.Encoding (decodeUtf8, encodeUtf8)
 import           Data.ByteString.Lazy (toStrict)
 
+import           Prelude -- AMP compatibility workaround
```

Comment:
[Explicit imports](https://github.com/ambiata/guideline/blob/master/language/haskell.md#imports) sound like a better solution to me if we can get that to work